src/app/children/notes/notes-manager/notes-manager.component.ts

```diff
@@ -140,6 +140,10 @@ export class NotesManagerComponent implements OnInit, AfterViewInit {
 
 
   showDetails(entity: Note) {
-    this.dialog.open(NoteDetailsComponent, {width: '80%', data: {entity: entity}});
+    const nextNote = this.dialog.open(NoteDetailsComponent, {width: '80%', data: {entity: entity}});
+
+    nextNote.afterClosed().subscribe(val => {
+      this.notesDataSource.data = [val].concat(this.entityList);
```

Comment:
also, because the class property `this.entityList` is used as the starting point whenever the user changes filters, we should add the new item to this so it doesn't disappear again suddenly.

This seems to work for me:
```
nextNote.afterClosed().subscribe(val => {
      this.entityList.push(val);
      this.applyFilterSelections();
});
```

Reply:
fixed.